test/package.json

```diff
@@ -8,7 +8,7 @@
 		"start": "mocha"
 	},
 	"dependencies": {
-		"webgl-raub": "https://github.com/node-3d/webgl-raub.git",
+		"webgl-raub": "1.3.0",
```

Comment:
should test against the current codebase, better remove this entry altogether and in test/index.js use require('..')